settings/version-number

```diff
@@ -1 +1 @@
-2.6.0
+2.7.0-alpha
```

Comment:
Should be changed?

Reply:
Yep, will be the last commit before the merge, as the ci will fail after it ^^